app/controllers/courses_controller.rb

```diff
@@ -42,10 +38,9 @@ def update
   end
 
   def duplicate
-    puts "What the fuck?"
```

Comment:
Why you remove this?

Reply:
In my view, it was a less-than-essential part of the code